### PR TITLE
Create an liveness server replying to healthz path

### PR DESCRIPTION
**Pull request by pipo02mix**
Towards https://github.com/giantswarm/giantswarm/issues/4224

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[![CircleCI](https://circleci.com/gh/giantswarm/liveness.svg?&style=shield&circle-token=4a9231c3fd0adf22193a186149ccaeb3a72188d4)](https://circleci.com/gh/giantswarm/liveness)
+
+# Liveness
+Application for checking the health of a cluster.
```
Comment (xh3b4sd) [inline README.md]: Tenant cluster, control plane, or both?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	ctx := context.Background()
+	mux := http.NewServeMux()
+	logger, err := micrologger.New(micrologger.Config{})
+	if err != nil {
+		panic(err)
+	}
+
+	mux.Handle("/healthz", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Write([]byte(""))
+	}))
```
Comment (xh3b4sd) [inline main.go:41]: What kind of liveness are we actually trying to verify here? Looks a bit similar to https://github.com/giantswarm/e2e-app. Where is the difference though?

**Review by xh3b4sd — COMMENTED** [review]
I would like to see different PRs like: 

* Circle setup. 
* Docker setup
* Helm setup. 
* Go app.

File: README.md (hunk 1 above)
Comment (pipo02mix) [reply README.md]: tenant (fix it)

File: main.go (hunk 2 above)
Comment (pipo02mix) [reply main.go:41]: yeah it was my inspiration. The idea is to enable istio ingress as main IC on KVM envs. 
For nginx deployments, there is a `/healthz` endpoint returning `200`. In istio we need to configure an app (and a virtual service and gateway) to verify it. I could use it the e2e app but it has different purpose and we can have conflicts. For example, liveness cannot return content in that endpoint or prometheus will fail to parse the response payload.

**Comment by pipo02mix** [comment]
Providing separates PRs

**Comment by pipo02mix** [comment]
Closing in favor of the small PRs

**Pull request merged**